radix-engine/src/vm/wasm/wasmi.rs

```diff
@@ -833,42 +797,44 @@ macro_rules! linker_define {
 
 #[derive(Debug)]
 pub enum WasmiInstantiationError {
-    ValidationError(Error),
+    CompilationError(Error),
     PreInstantiationError(Error),
     InstantiationError(InstantiationError),
 }
 
 impl WasmiModule {
     pub fn new(code: &[u8]) -> Result<Self, WasmiInstantiationError> {
-        let engine = Engine::default();
-        let mut store = Store::new(&engine, WasmiInstanceEnv::new());
-
-        let module =
-            Module::new(&engine, code).map_err(WasmiInstantiationError::ValidationError)?;
-
-        let instance = Self::host_funcs_set(&module, &mut store)
-            .map_err(WasmiInstantiationError::PreInstantiationError)?
-            .ensure_no_start(store.as_context_mut())
-            .map_err(WasmiInstantiationError::InstantiationError)?;
+        let mut config = wasmi::Config::default();
+
+        // In order to speed compilation we deliberately
```

Comment:
Note from the author.

Maybe it would make sense to setup this by extending `WasmiEngineOptions` with respective fields.
And then we could add more setup methods next to `DefaultWasmEngine::default()`.
eg. 
- `DefaultWasmEngine::with_no_validation()` - which would become `DefaultWasmEngine::default()`
- `DefaultWasmEngine::with_validation()`

Just a loose thought. 

We would just need to remember about `wasmer` and find adequate settings for it.

Reply:
`wasmer` support has just been removed :)